modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,379 @@
+
+```
+SELECT * FROM demo.redpanda.ClickEvent;
+```
+
+== Access data in Iceberg tables
+
+You can use the same analytical tools to access table data in a data lake as you would for a relational database.
+
+For example, suppose you produce the same stream of events to a topic `ClickEvent`, and another topic `ClickEvent_schemaless`. The topics have Tiered Storage configured to an AWS S3 bucket. A sample record contains the following data:
+
+[,bash,role=no-copy]
+----
+{"user_id": 2324, "event_type": "BUTTON_CLICK", "ts": "2024-11-25T20:23:59.380Z"}
+----
+
+When you point your Iceberg-compatible tool or framework to the object storage location of the Iceberg tables, how you consume the data depends on whether you've registered a schema on which to derive the table format. In either approach, you do not need to rely on complex ETL jobs or pipelines to consume real-time data from Redpanda.
+
+=== Structured approach with schema
+
+Register the following schema for `ClickEvent` under the `ClickEvent-value` subject:
+
+[,avro]
+----
+{
+    "type" : "record",
+    "namespace" : "com.redpanda.examples.avro",
+    "name" : "ClickEvent",
+    "fields" : [
+       { "name": "user_id", "type" : "int" },
+       { "name": "event_type", "type" : "string" },
+       { "name": "ts", "type": "string" }
+    ]
+ }
+----
+
+[,bash]
+----
+rpk registry schema create ClickEvent-value --schema path/to/schema.avsc --type avro
+----
+
+Query engines such as Spark SQL provide Iceberg integrations to allow easy access to existing Iceberg tables in object storage. The table structure is derived from the schema. In this example, the query returns values from columns in the `ClickEvent` table, with the column names matching the schema fields:
+
+[,sql]
+----
+SELECT user_id, 
+  event_type, 
+  ts 
+FROM ClickEvent;
+----
+
+[,bash,role=no-copy]
+----
++---------+--------------+--------------------------+
+| user_id | event_type   | ts                       |
++---------+--------------+--------------------------+
+| 2324    | BUTTON_CLICK | 2024-11-25T20:23:59.380Z |
++---------+--------------+--------------------------+
+----
+
+=== Schemaless approach
+
+You can also forgo using a schema, which means using semi-structured data in Iceberg. 
+
+This example queries the semi-structured data in the `ClickEvent_schemaless` table, which consists of a column `redpanda` containing the record key, value, and metadata:
+
+[,sql]
+----
+SELECT 
+  redpanda.data
+FROM ClickEvent_schemaless;
+----
+
+[,bash,role=no-copy]
+----
++------------------------------------------------------------------------------+
+| redpanda.data                                                                |
++------------------------------------------------------------------------------+
+| {"user_id":2324,"event_type":"BUTTON_CLICK","ts":"2024-11-25T20:23:59.380Z"} |
++------------------------------------------------------------------------------+
+----
+
+== Suggested reading
```

Comment:
@kbatuigas  we need to make it clear that this example assumes use of the REST catalog type and that the table metadata has been transferred to the external catalog.  Otherwise this example will not work without a CREATE TABLE

Reply:
Correct.